nilearn/input_data/nifti_spheres_masker.py

```diff
@@ -138,6 +220,51 @@ def __call__(self, imgs):
         return signals, None
 
 
+class _InversionFunctor(object):
```

Comment:
Sorry if I misunderstood, but what is the point in making this a class rather than a function ?

Reply:
In many cases I was just following the extraction function - so there might be some redundancy here.